android/java/org/chromium/chrome/browser/brave_news/BraveNewsUtils.java

```diff
@@ -126,11 +128,18 @@ public static void logFeedItem(FeedItemsCard items, String id) {
         }
     }
 
-    public static boolean shouldDisplayNews() {
+    public static boolean shouldDisplayNewsFeed() {
         return BravePrefServiceBridge.getInstance().getShowNews()
                 && BravePrefServiceBridge.getInstance().getNewsOptIn();
     }
 
+    public static boolean shouldDisplayNewsOptin() {
+        return BravePrefServiceBridge.getInstance().getShowNews()
```

Comment:
we should not use `BravePrefServiceBridge` for user prefs. @tapanmodh can you create a follow up issue for it ?

Reply:
it is to get same pref as desktop. sure, I'll create that

Reply:
Follow up issue: https://github.com/brave/brave-browser/issues/33464